custom_model_runner/datarobot_drum/drum/custom_tasks/fit_adapters/classification_labels_util.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright 2022 DataRobot, Inc. and its affiliates.
+All rights reserved.
+This is proprietary source code of DataRobot, Inc. and its affiliates.
+Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+import sys
+
+import numpy as np
+import pandas as pd
+
+from datarobot_drum.drum.enum import TargetType
+from datarobot_drum.drum.exceptions import DrumCommonException
+
+
+def needs_class_labels(
+    target_type, negative_class_label=None, positive_class_label=None, class_labels=None
+):
+    if target_type == TargetType.BINARY:
+        return negative_class_label is None or positive_class_label is None
+    if target_type == TargetType.MULTICLASS:
+        return class_labels is None
+    return False
+
+
+def possibly_intuit_order(
```

Comment:
what does this do?  it seems like a lot of different things

Reply:
Returns class labels sorted by name, if the target type implies there are class labels.

Reply:
is it only used below?

Reply:
yep